src/api.js

```diff
@@ -102,6 +102,12 @@ var schema = function(validator, options) {
   return function(req, res, next) {
     // If input schema is defined we need to validate the input
     if (options.input !== undefined && !options.skipInputValidation) {
+      if (req.headers['content-type'] !== 'application/json') {
+        return res.status(400).json({
+          message: "Payload must be JSON with content-type: application/json " +
+                   "got content-type: " + (req.headers['content-type'] || null),
+        });
+      }
```

Comment:
I like this.  I also like that the error message is not crappy!  Do we need to worry about case of the headers?  For the s3-copy-proxy stuff, I'm using a library called Caseless to help with headers... Maybe we need something like that here?